### PR TITLE
Remove jsdoc, add tsdoc config file

### PR DESCRIPTION
**Pull request by fredj**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,6 @@
 *.pyc
 .*.swp
+apidoc/
```
Comment (sbrunner) [inline .gitignore]:
```suggestion
/apidoc/
```

File: tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{
+    "compilerOptions": {
+      /* Basic Options */
+      "target": "ES6",
+      "module": "system",
+      "lib": ["es2017", "dom"],
+      "allowJs": true,
+      "checkJs": false,
```
Comment (sbrunner) [inline tsconfig.json]: Shouldn't be true?

File: tsconfig.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+      "noEmit": true
+    },
+    "include": [
+      "src/**/*.js",
+      "contribs/gmf/src/**/*.js",
+      "node_modules/@types/openlayers/*",
+      "types/**/*"
```
Comment (sbrunner) [inline tsconfig.json]: Those tow lines are relay needed, wanted?

File: tsconfig.json (hunk 2 above)
Comment (fredj) [reply tsconfig.json]: the doc generation doesn't work if this option sis set to true (too many errors)

**Pull request merged**